### PR TITLE
Add fake data file

### LINKED ISSUE #30: Fake Data
Create a json file that have fake data for the Articles and Doctors.

### PR DESCRIPTION
**Pull request by eng-somaia**

### CODE REVIEW

**Comment by MuhannedNoman** [comment]
@eng-somaia Looks like the test has failed. Can you check that out please, and then tag me again once done.

**Comment by eng-somaia** [comment]
> @eng-somaia Looks like the test has failed. Can you check that out please, and then tag me again once done.

@MuhannedNoman  Passing tests.

File: fakeData.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
+const psychiarists = [
+    {
+        doctorImg: "https://images.all-free-download.com/images/graphiclarge/doctor_work_painting_colored_cartoon_character_sketch_6847622.jpg",
```
Comment (MuhannedNoman) [inline fakeData.js]: Let's edit this to image only.

File: fakeData.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        name: "Dr. Nabeel No'man",
+        specialtyies: "Anxiety, Bipolar Disorder, Depression, General Adult Psychiatry, Personality Disorders",
```
Comment (MuhannedNoman) [inline fakeData.js]: From the design looks like we will only display one thing. So Let's reflect that her and just pick one thing to add.

File: fakeData.js [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+        workplace: "Azal Hospital",
+        address: "60th street",
+        email: "example@gmail.com",
+        phone: "+967770000000"
+    }
+];
+const articles = [
+    {
```
Comment (MuhannedNoman) [inline fakeData.js]: We need to add date here since it is displayed in the Article page and in home page.

File: fakeData.js [hunk 4]
```diff
@@ -0,0 +1,93 @@
+const psychiarists = [
```
Comment (MuhannedNoman) [inline fakeData.js]: Let's add an ID for each element here.

Comment (MuhannedNoman) [reply fakeData.js]: Article included.

**Review by MuhannedNoman — CHANGES_REQUESTED** [review]

**Comment by MuhannedNoman** [comment]
Impressive work here @eng-somaia just left you some comments. Let me know what you think !

**Comment by MuhannedNoman** [comment]
One last thing let's add a folder in the `src` folder and call it `fake[object]Services.js` ex `fakeDoctorsServices.js`
Inside the file we will only have Doctor's  data. We will use ES6 Export keyword without the word module. Also, add a function and export it call it `get[Object]` that will return the whole data. 
This way we can add and edit stuff in this single file in the future. 
Ex
```
export const doctors = [{},{},{}]

export function getDocotrs() {
     return doctors.filter(d => d)
}

```

**Comment by eng-somaia** [comment]
@MuhannedNoman Edit all the things you mentioned above. Let me know if there is anything to add or edit.

**Review by MuhannedNoman — CHANGES_REQUESTED** [review]
LGTM, one thing though. 
Change `fakeData.js` to `fakeArticleService.js` and move the two files into a folder called `services` inside the `src` folder. Once that is implemented feel free to merge.

**Comment by eng-somaia** [comment]
Done.

**Pull request merged**